e2e/cypress/integration/enterprise/teams/search_teams_spec.js

```diff
@@ -40,8 +40,8 @@ describe('Search teams', () => {
         const displayName = uuidv4();
 
         // # Create a new team.
-        cy.apiCreateTeam('team-search', displayName).then((response) => {
-            this.createdTeamIDs.push(response.body.id);
+        cy.apiCreateTeam('team-search', displayName).then(({team}) => {
```

Comment:
Wouldn't using just `team` work instead of `{team}`? Similarly using `teams` instead of `{teams}`

Reply:
That will work too. Yes, it could simply return [`response.body`](https://github.com/mattermost/mattermost-webapp/pull/5806/files#diff-bc91d52642ea6ce4595a81941caa05a6L60) and use directly by `.then((name-anything) => {..})` but we'll always second guess on what exactly it's being returned. `response.body` could be anything depending on API call.
The [new approach is being explicit](https://github.com/mattermost/mattermost-webapp/pull/5806/files#diff-bc91d52642ea6ce4595a81941caa05a6R48) on what it's returning and documenting as much as possible in `*.d.ts` for better dev experience.